packages/studio-base/src/components/SettingsTreeEditor/index.tsx

```diff
@@ -4,28 +4,30 @@
 
 import ClearIcon from "@mui/icons-material/Clear";
 import SearchIcon from "@mui/icons-material/Search";
-import { AppBar, IconButton, TextField, styled as muiStyled } from "@mui/material";
+import { AppBar, IconButton, TextField } from "@mui/material";
 import memoizeWeak from "memoize-weak";
 import { useMemo, useState } from "react";
 import { DeepReadonly } from "ts-essentials";
+import { makeStyles } from "tss-react/mui";
 
 import { SettingsTree } from "@foxglove/studio";
 import Stack from "@foxglove/studio-base/components/Stack";
 
 import { NodeEditor } from "./NodeEditor";
 import { filterTreeNodes, prepareSettingsNodes } from "./utils";
 
-const StyledAppBar = muiStyled(AppBar, { skipSx: true })(({ theme }) => ({
-  top: -1,
-  zIndex: theme.zIndex.appBar - 1,
-  borderBottom: `1px solid ${theme.palette.divider}`,
-  padding: theme.spacing(1),
-}));
-
-const FieldGrid = muiStyled("div", { skipSx: true })(({ theme }) => ({
-  display: "grid",
-  gridTemplateColumns: "minmax(4rem, 1fr) minmax(4rem, 12rem)",
-  columnGap: theme.spacing(1),
+const useStyles = makeStyles()((theme) => ({
+  appBar: {
+    top: -1,
+    zIndex: theme.zIndex.appBar - 1,
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    padding: theme.spacing(1),
+  },
+  fieldGrid: {
+    display: "grid",
+    gridTemplateColumns: "minmax(0, 1fr) minmax(0, 2fr)",
```

Comment:
Is there any reason why we wouldn't want to do this?

```suggestion
    gridTemplateColumns: "minmax(0, max-content) minmax(0, 2fr)",
```

Reply:
Yeah this means long labels will be shown in full width without any truncation. Maybe that's ok and we just ask people to be reasonable with the length of labels?